### ISSUE TITLE
Content does not show for <Tabs>

### ISSUE THREAD

**Issue by ckim16** [GeekyAnts/NativeBase#878]
Hi,

I'm using `"native-base": "^2.1.3"` and `"react-native": "0.44.0"` and the content for my tabs does not show and I'm not sure how to fix this.

```
import HomeTab from '../components/homeTab';

render() {
    return (
      <Drawer
        ref={(ref) => { this._drawer = ref; }}
        content={<Sidebar navigator={this._navigator}/>}
        onClose={() => this.closeDrawer()}
        >
        <Container>        
          <Header style={styles.header} hasTabs>
            <Tabs tabBarUnderlineStyle={styles.underLine}>
              <Tab
                activeTabStyle={styles.header}
                activeTextStyle={styles.text}
                tabStyle={styles.header}
                textStyle={styles.text}
                heading="Home"
                >
                <HomeTab />
              </Tab>
              <Tab
                activeTabStyle={styles.header}
                activeTextStyle={styles.text}
                tabStyle={styles.header}
                textStyle={styles.text}
                heading="Something else"
                >
                <Button><Text>Hello?</Text></Button>
              </Tab>
            </Tabs>
          </Header>
        </Container>
      </Drawer>
    );
  }
}
```

I just want to see something showing under the tab bar but nothing really appears on both Home tab and something else tab. And only thing that HomeTab contains is
```
render() {
    return (
      <Container>
        <Content>
          <Text>This is Home Tab</Text>
        </Content>
      </Container>
    );
  }
```

Has anyone had a same issue that I am going through and know how to fix this?

Thank you all for your time.

**Comment by SupriyaKalghatgi** [comment]
@ckim16 The first check you need to do is

NativeBase 2.1.4 says RN >= 0.44.0; and your package.json says `NB: ^2.1.3` and `RN: 0.44.0`
I suggest you to check the version of NativeBase from `node_modules`

**Comment by ckim16** [comment]
I updated my NB to `"^2.1.4"` and RN to `"^0.44.2"` but still does not work.

Any other debug I can do to make this work?

**Comment by yaronlevi** [comment]
happens to me as well

**Comment by yaronlevi** [comment]
An update- the issue is related to the phone system language being an RTL. My language is Heb. If I set it back to Eng, the issue disappears.

**Comment by ckim16** [comment]
What about running in the simulator? I'm using the Xcode and it still does not work for me :(

**Comment by SupriyaKalghatgi** [comment]
@ckim16 After updating NB to `"^2.1.4"` and RN to `"^0.44.2"`
You find issue with Tabs only or with other components as well?
Also that, could you run this successfully on simulator and that the issue is while running on device?

**Comment by SupriyaKalghatgi** [comment]
@yaronlevi Just this component?

**Comment by ckim16** [comment]
@SupriyaKalghatgi Right now my only issue is Tabs component and I've been only using simulator so far. There is another issue where I cannot use 'smoking' icon from  react-native-vector-icons but I will open a new issue after I try little bit more.

**Comment by SupriyaKalghatgi** [comment]
@ckim16 You should move the `<Tabs>` outside `<Header>`
Check the example from [NativeBase-KitchenSink](https://github.com/GeekyAnts/NativeBase-KitchenSink)

**Comment by ckim16** [comment]
Oh, that is what it was. Thank you so much for catching that for me. I really appreciate it!

**Comment by saivyas** [comment]
I had problem with native base tabs when I did forceRtl. Everything works but only native base doesn't show anything on initial load .

So I had a temporary solution

@ https://stackoverflow.com/questions/58736619/nativebase-tabs-issue-for-force-rtl-fix/58736621#58736621
